db/schema.rb

```diff
@@ -98,4 +98,14 @@
     t.index ["default_ccli_user_id"], name: "index_users_on_default_ccli_user_id"
   end
 
+  create_table "versions", force: :cascade do |t|
```

Comment:
I am not entirely sure why you have the versions table here.
Probably a left over from working on the other branches?
As you are basing this branch on master which currently doesn't have any versions this would probably break some stuff :joy: unless I am misunderstanding something :angel:

Reply:
Whoops! yes this is a mistake 😆